programmers/난이도별/level01.부족한_금액_계산하기/JiaLee0707.js

```diff
@@ -0,0 +1,10 @@
+function solution(price, money, count) {
+    let answer = 0;
+    for(let i=1; i<=count; i++) {
+        answer += price*i;
+    }
+
+    answer = (answer >= money ? answer -= money : 0);
+    
+    return answer;
```

Comment:
![image](https://user-images.githubusercontent.com/48716298/159599839-cd3329c5-8bf7-4eae-b939-6396075dc242.png)
이 문제는 더 파고 들어가면 **등차수열의 합**을 구하는 문제라고 보여져요!
테스트 케이스에 있는 데이터로 말씀드리면 `3`이라는 처음 이용료에 일정한 간격의 이용료가 놀이기구 탄 횟수만큼 더해지는거죠! `=> 3+6+9+12` 

등차수열의 공식은 [참고 링크](https://mathbang.net/609)를 확인하였고 코드를 아래와 같이 수정할 수 있을 것 같아요.
```suggestion
    const result = price * count * (count + 1) / 2 - money;
    return result > 0 ? result : 0;
```

Reply:
와! 코드를 더욱 간결하게 할 수 있는 방법이 있다니...
좋은 정보 정말 감사합니다!😆